query/stdlib/influxdata/influxdb/rules.go

```diff
@@ -31,7 +31,7 @@ func init() {
 		PushDownWindowAggregateByTimeRule{},
 		PushDownBareAggregateRule{},
 		GroupWindowAggregateTransposeRule{},
-		PushDownGroupAggregateRule{},
+		// PushDownGroupAggregateRule{},
```

Comment:
This PR does not include grouped aggregates. This rule still needs to be enabled and the corresponding launcher tests unskipped.

Reply:
Ok. By the way `first` and `last` were not supposed to be a part of this issue either. There were separate issues for them. I'm not complaining though, it's nice to get them out of the way.